### PR TITLE
All cppcheck config options, evaluate environment variables.

### PR DESCRIPTION
**Pull request by claudiiolima**
Remove old config option from doc.

### CODE REVIEW

**Comment by claudiiolima** [comment]
This PR solves #6

**Review by deltalejo — CHANGES_REQUESTED** [review]
I don't think it is needed to support environment variables evaluation for all config options, it would be better to select some sensible subset.
Also, bear in mind that not all options are a string, so as a safe guard it should be checked if the config value is a string and only then do the match comparison.

File: lib/cppcheck.rb [hunk 1]
```diff
@@ -9,11 +9,20 @@ def setup
...
     @loginator = @ceedling[:loginator]
     @setupinator = @ceedling[:setupinator]
+    @system_wrapper = @ceedling[:system_wrapper]
     @tool_executor = @ceedling[:tool_executor]
     @tool_validator = @ceedling[:tool_validator]
-
+    
     @config = @setupinator.config_hash[CPPCHECK_SYM]
     
+    @config.each do |key,value|
```
Comment (deltalejo) [inline lib/cppcheck.rb]: `value` is not used inside the block, it should be removed.

Comment (deltalejo) [reply lib/cppcheck.rb]: or may you want to use it for the match and as the argument for `module_eval()`?

Comment (claudiiolima) [reply lib/cppcheck.rb]: I did what you proposed, and the Array is also handled.

**Review by deltalejo — APPROVED** [review]

**Comment by deltalejo** [comment]
Thanks!

**Pull request merged**